web-app/src/components/tournament/GridListMobile.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="q-pa-md lt-sm">
+    <q-table
+      grid
+      title="Tournaments"
+      :data="data"
+      :columns="columns"
+      row-key="name"
+      :filter="filter"
+      virtual-scroll
+      :pagination="pagination"
+      :rows-per-page-options="[0]"
+      :expanded.sync="expanded"
+    >
+      <template v-slot:top-right>
+        <q-input
+          dense
+          debounce="300"
+          v-model="filter"
+          placeholder="Search"
+          style="width:150px"
+        >
+          <template v-slot:append>
+            <q-icon name="search" />
+          </template>
+        </q-input>
+      </template>
+
+      <template v-slot:item="props">
+        <div class="row q-pa-xs col-12">
+          <q-btn no-caps flat padding="none" class="col-12">
+            <q-card class="row col-12">
+              <q-card-section
+                class="col-6 overflow-hidden"
+                style="text-align:left"
+              >
+                <div class="text-grey-14">
+                  Name:
+                </div>
+                <strong>{{ props.row.name }}</strong>
+              </q-card-section>
+              <q-card-section
+                class="col-6"
+                style="text-align:right"
+                :style="
+                  props.row.status === 'In progress'
+                    ? 'color: orange'
+                    : 'color: green'
+                "
+              >
```

Comment:
Maybe better make this on divs, not q-card-sections. bcz you have to use overflow-hidden when you use col-6 (1:1) - a lot of space for status name
But this is a details again 
`[[Name, <name>] <status>]`